### PR TITLE
styles: general component improvements

### LINKED ISSUE #92: [Task]: Improve styles for HighlightCard
- [ ] Replace `height` style with `min-height`
- [ ] Reduce min-width for something a bit smaller. A min width of 330px only allows screens for over 378px before they start overflowing. 330px + 48 (24px per side padding). Reduce at least 18px so that at least 360px screens fit perfectly.
- [ ] max-width: 800px
- [ ] add missing isExternal prop when passing link

### LINKED ISSUE #93: [Task]: Allow line breaks in the `IconFeatures` and similar components

### LINKED ISSUE #94: [Task]: Fix `Title` sizes and tags

### LINKED ISSUE #97: [Task]: Allow poster prop for video in MediaManager

### PR DESCRIPTION
**Pull request by VmMad**
Fixes #94
Fixes #92 
Fixes #93 
Fixes #97

### CODE REVIEW

File: src/lib/components/atoms/title/title.constants.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ export const HEADING_TO_SIZE: Record<TitleTag, TitleSize> = {
 }
 
 export const SIZE_TO_HEADING: Record<TitleSize, TitleTag> = {
-    [TitleSize.Large]: TitleTag.H1,
-    [TitleSize.Medium]: TitleTag.H2,
-    [TitleSize.Small]: TitleTag.H3,
-    [TitleSize.ExtraSmall]: TitleTag.H4,
+    [TitleSize.Large]: TitleTag.H2,
```
Comment (evavirseda) [inline src/lib/components/atoms/title/title.constants.ts:13]: just one question, is removing the `h1` intentional? or maybe can we add `extraLarge` for it? :pray:

**Review by evavirseda — CHANGES_REQUESTED** [review]

File: src/lib/components/atoms/title/title.constants.ts (hunk 1 above)
Comment (VmMad) [reply src/lib/components/atoms/title/title.constants.ts:13]: Usually the websites only have one h1 and the hero component already uses h1, so if it's really necessary, you can just pass the tag to be H1. But if you only pass `size={TitleSize.Large}` then it just fallbacks to `TitleTag.H2` unless tag is specified.

...
This is just to make components easier to use, but I can add the H1 back if you want. Let me know @evavirseda

Comment (evavirseda) [reply src/lib/components/atoms/title/title.constants.ts:13]: it's ok, just curious :pray:

**Review by evavirseda — APPROVED** [review]
nice job :rocket:

**Review by evavirseda — APPROVED** [review]
:rocket:

File: src/lib/components/atoms/title/title.constants.ts (hunk 1 above)
Comment (begonaalvarezd) [inline src/lib/components/atoms/title/title.constants.ts]: wouldnt this break our already existing produciton websites like grants?

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**